Tokenizer/tokenizer.py

```diff
@@ -1,7 +1,7 @@
 '''
 Python implementation of Csq Tokenizer
 '''
-from ..Grammar.grammer import *
+from Grammar.grammar import *
```

Comment:
from ..Grammar.grammer import * 
This import statement uses relative import to access a module that is in a parent directory of the current module.
and i think we should use this

Reply:
The code is called by main.py file due to which relative import is not required and the 2nd  thing is that it's very much prone to errors i tried to use it but due to some sort of file sturct error relative import was blocked.

Reply:
okye.